### ISSUE TITLE
Implement Mollie webhook script in Cyclos

### ISSUE THREAD

**Issue by sandrab** [circuitnederland/registrationform#2]
There is a script of type custom web service in Cyclos called mollieWebhook. This is called by Mollie after a payment is done. The script is not yet fully implemented and needs some cleaning-up.

**Comment by rmvanarkel** [comment]
Please have a look at the documentation: https://documentation.cyclos.org/4.10/cyclos-reference/scripting-types.html#d0e4075 (the webhook should point to something like /run/<mollie webhook url part>). Please make sure to use a good strong password for production if possible (gast met toestemming) and use the ip whitelist if possible.

**Comment by sandrab** [comment]
Implemented a first version of the mollieWebhook script, see c885ea6668aa8c0c83753e4e26b18a9408a4c1c8. It contains a MailHelper class that could be moved to a library script - this helper class could be used more generically in other places where we need to send e-mails as well.
The helper class needs a parameter, with the e-mailaddress of the tech team. It is used to send e-mails indicating technical issues.
@rmvanarkel : Do you think it could be handy to make a separate mailaddress for this (tech@c3.nl)? Or is there a tech address we can use for this already?

**Comment by rmvanarkel** [comment]
Yes I think it would be handy to have a special email address for tech@circuitnederland.nl. Can you arrange this with Menno? And for now forward it to Guus his email address? Currently the email address is set in the molly configuration for what Rinke programmed.

**Comment by sandrab** [comment]
The mollieWebhook script is implemented by 48986d4 as part of the new registration flow.
